app/controllers/api/v1/search_skills_controller.rb

```diff
@@ -3,15 +3,26 @@ class Api::V1::SearchSkillsController < ApplicationController
   before_action :check_for_empty_query
 
   def index
```

Comment:
It might get a bit weird due to the structure of how @users are being generated, but if a json-api serializer is passed empty data, it just generates the empty array for you rather than having to do it as a separate condition.

Not that it's a problem the way you're doing it, just a heads up the serializer can actually handle it for you!